store/sqlstore/utils.go

```diff
@@ -89,3 +89,30 @@ func isQuotedWord(s string) bool {
 
 	return s[0] == '"' && s[len(s)-1] == '"'
 }
+
+// constructMySQLJSONArgs returns the arg list to pass to a query along with
+// the string of placeholders which is needed to be to the JSON_SET function.
+// Use this function in this way:
+// UPDATE Table
+// SET Col = JSON_SET(Col, `+argString+`)
+// WHERE Id=?`, args...)
+// after appending the Id param to the args slice.
+func constructMySQLJSONArgs(props map[string]string) ([]interface{}, string) {
+	if len(props) == 0 {
+		return nil, ""
+	}
+
+	// Unpack the keys and values to pass to MySQL.
+	args := make([]interface{}, 0, len(props))
+	for k, v := range props {
+		args = append(args, "$."+k)
+		args = append(args, v)
+	}
+
+	// We calculate the number of ? to set in the query string.
+	argString := strings.Repeat("?, ", len(props)*2)
+	// Strip off the trailing comma.
+	argString = argString[:len(argString)-2]
```

Comment:
nit:
```suggestion
	argString = strings.TrimSuffix(strings.TrimSpace(argString), ",")
```

Reply:
Ooh good one. I was pretty sure there was a neat way to do that, but didn't spend enough mental cycles :P